### ISSUE TITLE
2J5X0001.000 - checksum

### ISSUE THREAD

**Issue by kusala9** [kusala9/S-64#11]
Chart 2J5X0001.000 has wrong checksum in test data set.  We have noted that this chart has relative new time stamp as 17th Apr 2015.  Most probable the catalog.031 file contains checksum of the previous version of this chart cell

Fix the test data set
Note that the chart dated 23.04.2014 was ok and can be used for testing

**Comment by kusala9** [comment]
yep. Fixed this with commit b5b467ee7802a31199c6e4124bb05a2751a2e2ba

**Comment by kusala9** [comment]
added label for test.

**Comment by tmellor** [comment]
S-58 Tests removed unknown objects from dataset
[3.6 Display Priorities.zip](https://github.com/kusala9/S-64/files/1057506/3.6.Display.Priorities.zip)
Use original file from zip and correct the checksum again!
